### ISSUE TITLE
84mm HEDP rounds have incorrect stats, do not work as intended. 

### ISSUE THREAD

**Issue by T-rexsuper1** [cataclysmbnteam/Cataclysm-BN#2380]
### Describe the bug

Carl Gustaf M3 recoilless rifle HEDP ammo straight up does not work as HEDP ammo. 
The difference between HEDP and HE ammo is miniscule in BN. HE is supposed to be AoE damage with low AP but high explosion damage. And it works as intended. 
HEDP, for some reason, does the exact same. Maybe the explosion radius is different but the direct hit damage is not and it has very little damage and armor penetration. See the DDA version (pic related) to view the actual stats it is supposed to have. 

But what is a HEDP round? Quoting wikipedia (sources are cited)
FFV502 is an HEDP round with the ability to be set to detonate on either impact or one-tenth second later. Effective range is 1,000 m against dispersed soft targets such as infantry in the open, 500 m against stationary targets and 300 m against moving targets. Minimum range is 15 to 40 m to arm the warhead. Penetration exceeds 150 mm of rolled homogeneous armour (RHA). Ammunition weight is 3.3 kg and muzzle velocity is 230 m/s.

As you can see, it can penetrate up to 150mm of RHA. Needles to say that ingame stats of 10 AP do not come even close to this value. As for the damage, which DDA has 1000 of, it makes sense because it's an anti tank round designed to punch through and destroy anything behind.

In conclusion, pls port the stats from DDA :3

### Steps To Reproduce

Spawn or find an M3 HEDP round and look at the stats. 

And yes, it does not destroy a Beagle tank. Which it should. In fact, it should destroy anything with one hit, being an anti tank round. 

### Screenshots

![image](https://user-images.githubusercontent.com/30936470/221865522-03f63867-07ec-4e59-83b1-be5d01ae0514.png)
^^^^ BN stats 

![image](https://user-images.githubusercontent.com/30936470/221865621-819d7600-24cc-4c20-94ce-e471efc22769.png)
^^^^ DDA stats 

### Versions and configuration

Win10
Stable 0.2
Tiles UDP
English
No mods

**Comment by Izergaer** [comment]
yep. the damage is too low imo

**Comment by T-rexsuper1** [comment]
> yep. the damage is too low imo

by too low, it is 970 health points low. And the penetration is 215 points low. On direct hits of course.

**Comment by Coolthulhu** [comment]
Huge dmg items like that pose a balance problem. They are "too good to use", except when there's a supposedly difficult target that they instakill.

Rough balancing guidelines:
* If it instakills kevlar hulks on a graze, it doesn't exist
* If it instakills kevlar hulks on a crit, it's non-renewable and mounted on a vehicle
* If it can be expected to kill a regular hulk in one turn, it is either mounted on a vehicle or requires "mountable" terrain/furniture to use
